code/modules/mob/living/carbon/alien/humanoid/emote.dm

```diff
@@ -8,8 +8,23 @@
 //	if(findtext(act,"s",-1) && !findtext(act,"_",-2))//Removes ending s's unless they are prefixed with a '_'
 //		act = copytext(act,1,length(act))	//seriously who the fuck wrote this
 	var/muzzled = is_muzzled()
+
+	var/on_CD = 0
```

Comment:
Aaaah oh god I forgot how terribly confusing the emote CD system is. I'll have to clean this up some time later, if it works it works though.